examples/plotting/line_on_off.py

```diff
@@ -1,4 +1,11 @@
-""" Example demonstrating turning lines on and off - with JS only
+""" Example demonstrating turning lines on and off - with JS only.
+It involves checking and unchecking the checkboxes representing
+the plotted lines to turn the lines on/off.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.line, bokeh.layouts.row
+    :refs: :ref:`ug_basic_lines_line_multiple`
```

Comment:
Let's also add JS Callbacks here, and maybe the checkbox, while we are at it:
```suggestion
    :refs: :ref:`ug_interaction_js_callbacks`, :ref:`_ug_interaction_widgets_examples_checkboxgroup`, :ref:`ug_basic_lines_line_multiple`
```